src/server.h

```diff
@@ -563,13 +563,12 @@ typedef enum {
 #define PROPAGATE_AOF 1
 #define PROPAGATE_REPL 2
 
-/* Client pause types, larger types are more restrictive
- * pause types than smaller pause types. */
-typedef enum {
-    CLIENT_PAUSE_OFF = 0, /* Pause no commands */
-    CLIENT_PAUSE_WRITE,   /* Pause write commands */
-    CLIENT_PAUSE_ALL      /* Pause all commands */
-} pause_type;
+/* Services pause types */
+#define PAUSE_SVC_CLIENT_WRITE  (1<<0)
+#define PAUSE_SVC_CLIENT_ALL    (1<<1) /* must be bigger than PAUSE_SVC_CLIENT_WRITE */
+#define PAUSE_SVC_EXPIRE        (1<<2)
+#define PAUSE_SVC_EVICT         (1<<3)
+#define PAUSE_SVC_REPLICA       (1<<4) /* pause replica traffic */
```

Comment:
Might want to create an alias for PAUSE_SVC_ANY_MUTATION, that aliases all of these since that seems to be the most common one.

Reply:
We shouldn't configure PAUSE_SVC_CLIENT_WRITE along with PAUSE_SVC_CLIENT_ALL.
Prefer to leave it explicitly for now.

Reply:
why can't we set them both together?
also the ANY_MUTATION could be just set to WRITE, EXPIRE, EVICT and REPLICA (excluding ALL since it's not a mutation)